cfn-resources/cfn-publishing-automation/README.md

```diff
@@ -0,0 +1,43 @@
+# CFN publishing automation document for AWS System Manager
+
+** *Documents in this directory intended for internal use only by MongoDB, Inc.*
+
+cfn-3P-provider-register.yaml file contains the automation document that is used to publish
+MongoDB's CFN resources to AWS public registry.
+
+This automation document builds and registers 3rd party resource providers. This automation document
+can register resources in a single AWS account or in multiple accounts and regions. However,
+multi-account and regions require a role. Please refer to
+the [SSM multi-account ](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-automation-multiple-accounts-and-regions.html)
+documentation.
+
+This document builds and registers 3rd party resource providers build using the following plugins:
+
+* JAVA [link to another webpage](https://aws.amazon.com/)
+* Go [link to another webpage](https://aws.amazon.com/)
+* Python (Coming soon) [link to another webpage](https://aws.amazon.com/)
+
+## Making updates to the automation:
+
+- The .yaml script is a copy of "CFN-MongoDB-Atlas-Resource-Register" automation document that can
+  be found in AWS Systems Manager (please reach out to team for access)
+
+To update the script in AWS you will need to create a new version of the document. We aim to keep a version in source control at all times which matches the currently deployed version. Please follow
+below instructions:
+
+1. Create a new test document in AWS account with your updated script and test your changes.
```

Comment:
Could you add a link to the AWS doc on how to add this new document?

Reply:
added link